packages/open-payments/src/client/incoming-payment.test.ts

```diff
@@ -64,7 +66,7 @@ describe('incoming-payment', (): void => {
         },
         openApiValidators.successfulValidator
       )
-      expect(result).toStrictEqual(incomingPayment)
+      expect(result).toEqual(incomingPayment)
```

Comment:
Can you elaborate on why this was necessary? I changed back to `toStrictEqual` and it fails with `serializes to the same string`. Something that changed with nock? or?

Reply:
I thought this was odd as well, but I think it's the difference between how a response is returned in axios vs fetch. It's possible the order in which the object keys are returned are different?

Someone has an unaswered question related exactly to this: https://stackoverflow.com/questions/76211440/why-does-jests-tostrictequal-fail-to-match-json-responses-made-using-the-fetch